scripts/ansible/roles/linux/openenclave/tasks/ubuntu/lvi-setup.yml

```diff
@@ -41,12 +41,13 @@
 
 - name: Download as/ld tarball
   get_url:
-    url: 'https://download.01.org/intel-sgx/sgx-linux/2.10/as.ld.objdump.gold.r2.tar.gz'
-    dest: '/tmp/as.ld.objdump.gold.r2.tar.gz'
+    url: 'https://download.01.org/intel-sgx/sgx-linux/2.13/as.ld.objdump.gold.r3.tar.gz'
```

Comment:
@mingweishih Does clang based LVI mitigation need these ld binaries? Or can we get rid of them since we no longer build the SDK with GCC?

Reply:
Yes, even clang still needs those ld/as binaries as the binary instrumentation is done by the assembler. We still pass `no-integrated-as` to the clang when enabling LVI mitigation.

We may be able to drop these dependencies once we update to the clang-10 or 11 which supports LVI mitigation.

Reply:
>once we update to the clang-10 or 11 which supports LVI mitigation.

Can you confirm this? There would be an interest in simplifying these scripts (not within this PR) and these dependencies are the only ones that get us into trouble when we need to chicken-egg it.

Reply:
The feature seems lands on clang-12. I guess we still need to keep these dependencies around for a while 😄